pkg/v1/google/list_test.go

```diff
@@ -67,6 +67,64 @@ func TestRoundtrip(t *testing.T) {
 	}
 }
 
+// GCR returns timeUploaded as string
+func TestTimeUploadedMsAsString(t *testing.T) {
+	data := []byte(`
+		{
+			"imageSizeBytes": "100",
```

Comment:
Indentation is off here and below.

Reply:
Oops, should be fixed now, thanks!
Please let me know if you want me to squash the commits.